### PR TITLE
chore: Update develop with changes from release of 2.5.0

### PR DESCRIPTION
**Pull request by fuzzybinary**
### What and why?

Changes related to releasing `datadog_flutter_plugin` 2.5.0

### Review checklist

- [ ] This pull request has appropriate unit and / or integration tests 
- [ ] This pull request references a Github or JIRA issue

### CODE REVIEW

File: packages/datadog_flutter_plugin/CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,21 @@
 
 ## Unreleased
 
+
+
+## 2.5.0
+
 * Support 128-bit trace ids in distributed tracing.
+* Update iOS to 2.11.0. For a full list of changes see the [iOS Changelog](https://github.com/DataDog/dd-sdk-ios/blob/develop/CHANGELOG.md#2110--08-05-2024)
```
Comment (rtrieu) [inline packages/datadog_flutter_plugin/CHANGELOG.md]:
```suggestion
* Update iOS to 2.11.0. For a full list of changes, see the [iOS changelog](https://github.com/DataDog/dd-sdk-ios/blob/develop/CHANGELOG.md#2110--08-05-2024).
```

File: packages/datadog_flutter_plugin/CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,21 @@
...
+  * Add Fatal App hang tracking in RUM.
```
Comment (rtrieu) [inline packages/datadog_flutter_plugin/CHANGELOG.md]:
```suggestion
  * Add fatal app hang tracking in RUM.
```

File: packages/datadog_flutter_plugin/CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,21 @@
...
+  * Solve false-positive Privacy Manifest warnings on Required Reason API usage.
```
Comment (rtrieu) [inline packages/datadog_flutter_plugin/CHANGELOG.md]:
```suggestion
  * Solve false-positive privacy manifest warnings on Required Reason API usage.
```

File: packages/datadog_flutter_plugin/CHANGELOG.md [hunk 4]
```diff
@@ -2,7 +2,21 @@
...
+  * Call RUM's `errorEventMapper` for crashes.
+  * Support calling log event mapper for crashes
```
Comment (rtrieu) [inline packages/datadog_flutter_plugin/CHANGELOG.md]:
```suggestion
  * Support calling log event mapper for crashes.
```

File: packages/datadog_flutter_plugin/CHANGELOG.md [hunk 5]
```diff
@@ -2,7 +2,21 @@
...
+  * Start sending data immediately after SDK is initialized.
+* Update Android to 2.9.0. For a full list of changes see the [Android Changelog](https://github.com/DataDog/dd-sdk-android/blob/develop/CHANGELOG.md#290--2024-05-02)
+  * Call RUM's `errorEventMapper` for crashes.
+  * Support calling log event mapper for crashes
```
Comment (rtrieu) [inline packages/datadog_flutter_plugin/CHANGELOG.md]:
```suggestion
  * Support calling log event mapper for crashes.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Some feedback for style consistency.

File: packages/datadog_flutter_plugin/CHANGELOG.md [hunk 6]
```diff
@@ -2,7 +2,21 @@
...
 * Support 128-bit trace ids in distributed tracing.
+* Update iOS to 2.11.0. For a full list of changes, see the [iOS changelog](https://github.com/DataDog/dd-sdk-ios/blob/develop/CHANGELOG.md#2110--08-05-2024).
+  * Add fatal app hang tracking in RUM.
+  * Solve false-positive privacy manifest warnings on Required Reason API usage.
+  * Call RUM's `errorEventMapper` for crashes.
+  * Support calling log event mapper for crashes.
+  * Start sending data immediately after SDK is initialized.
+* Update Android to 2.9.0. For a full list of changes see the [Android Changelog](https://github.com/DataDog/dd-sdk-android/blob/develop/CHANGELOG.md#290--2024-05-02)
+  * Call RUM's `errorEventMapper` for crashes.
+  * Support calling log event mapper for crashes.
```
Comment (0xnm) [inline packages/datadog_flutter_plugin/CHANGELOG.md]: this was there since 2.0.0.

**Review by 0xnm — DISMISSED** [review]

File: packages/datadog_flutter_plugin/CHANGELOG.md (hunk 6 above)
Comment (fuzzybinary) [reply packages/datadog_flutter_plugin/CHANGELOG.md]: Both or just the logger?

Comment (0xnm) [reply packages/datadog_flutter_plugin/CHANGELOG.md]: only for logger

**Review by 0xnm — APPROVED** [review]

**Review by rtrieu — APPROVED** [review]
LGTM

**Pull request merged**